lib/howl/ui/editor.moon

```diff
@@ -989,6 +1002,8 @@ for cmd_spec in *{
   { 'scroll-up', 'Scrolls one line up', 'scroll_up' }
   { 'scroll-down', 'Scrolls one line down', 'scroll_down' }
   { 'duplicate-current', 'Duplicates the selection or current line', 'duplicate_current' }
+  { 'cut', 'Cut the selection or current line to the clipboard', 'cut' }
```

Comment:
Final nitpick - the descriptions should be fixed to be in the same form as the other descriptions, e.g. "Cuts the", "Copies the". After that we're good to go :)